crates/zizmor/src/main.rs

```diff
@@ -142,6 +143,48 @@ struct App {
     inputs: Vec<String>,
 }
 
+/// Shell with auto-generated completion script available.
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq, ValueEnum)]
+#[allow(clippy::enum_variant_names)]
+enum Shell {
+    /// Bourne Again `SHell` (bash)
+    Bash,
+    /// Elvish shell
+    Elvish,
+    /// Friendly Interactive `SHell` (fish)
+    Fish,
+    /// Nushell
+    Nushell,
+    /// `PowerShell`
+    PowerShell,
```

Comment:
I think if you drop the case here this will become `powershell` instead of `power-shell` in the `ValueEnum` 🙂 

```suggestion
    Powershell,
```

(That will also allow us to remove the `enum_variant_names` annotation.)